studio-docs/source/explorer.mdx

```diff
@@ -1,16 +1,36 @@
 ---
 title: The Apollo Studio Explorer
-sidebar_title: The Explorer
+sidebar_title: Explorer & Sandbox
 ---
 
 import { VideoFrame } from 'gatsby-theme-apollo-docs'
+import { Button } from '@apollo/space-kit/Button';
+import { Link } from 'gatsby';
+import { colors } from 'gatsby-theme-apollo-core';
 
 The Apollo Studio Explorer is a powerful web IDE for creating, running, and managing GraphQL operations:
 
 <VideoFrame src="https://www.youtube.com/embed/j8b0Bda_TIw"/>
 
 The Explorer is free for all Apollo Studio organizations. It supports all GraphQL operation types (`Query`, `Mutation`, and `Subscription`).
 
+## Account-free sandbox
+
+**Apollo Sandbox** is a special instance of the Explorer that doesn't require an Apollo account. It doesn't support advanced features like schema history, but it's great for local development.
+
+<p>
+  <Button
+    size="large"
+    color={colors.primary}
+    as={<Link to="https://studio.apollographql.com/sandbox/" target="_blank" />}
```

Comment:
Is there a placeholder endpoint we could point people to for demo purposes? (e.g. acephi? Blade's countries endpoint?)